doc/building-plugins.dox

```diff
@@ -227,7 +227,7 @@ enable them, do the following:
 
 -   For @ref Trade::BasisImporter "BasisImporter" or
     @ref Trade::BasisImageConverter "BasisImageConverter", [download the
-    `v1_15_update2` tag of the Basis Universal repo](https://github.com/BinomialLLC/basis_universal/archive/v1_15_update2.tar.gz),
+    `v1_50_0_2` tag of the Basis Universal repo](https://github.com/BinomialLLC/basis_universal/archive/v1_50_0_2.tar.gz),
```

Comment:
Since it should work with 1.15 and 1.16 as well, I'd mention those tags too, but give a link only to the latest and greatest.

Same in the plugin docs, mention that it's tested with all three.